### PR TITLE
feat: Add TemplateWizard.json to .template.config of unoapp template

### LINKED ISSUE #895: Migrate TemplateWizard.json
## What would you like to be added:

Migrate the TemplateWizard.json to the UnoApp template

## Why is this needed:

This is currently duplicated by both the Rider and Visual Studio extensions. This will ensure both remain in sync with the Template.

### PR DESCRIPTION
**Pull request by Takoooooo**
TemplateWizard.json is the file which is used by the Uno team internally to describe the dependencies of unoapp parameters. We need to move this file here becuse we use it internally in to different repos and in this way we could deduplicate the use of TemplateWizard.json

Fixes #895

### CODE REVIEW

**Comment by Takoooooo** [comment]
@dansiegel  @nickrandolph

**Comment by agneszitte** [comment]
@Takoooooo, @dansiegel is this PR needs to be merged before the release for the new Uno Platform plugin for Rider EAP ?
(cf. https://github.com/unoplatform/uno.rider/issues/79)

**Comment by Takoooooo** [comment]
> @Takoooooo, @dansiegel does this PR needs to be merged before the release of the new Uno Platform plugin for Rider EAP ? (cf. [unoplatform/uno.rider#79](https://github.com/unoplatform/uno.rider/issues/79))

No, TemplateWizard.json is already embedded into plugin. I am just wanting to move it to uno.templates because it is duplicated between uno.studio and uno.rider

File: src/Uno.Templates/content/unoapp/.template.config/TemplateWizard.json [hunk 1]
```diff
@@ -0,0 +1,707 @@
...
+			"Description": "Select the continuous integration files to be created"
+		}
+	],
+	"TemplateTypes": {
+		"SymbolId": "preset",
+		"Choices": [
+			{
+				"Id": "blank",
+				"Sequence": 1,
+				"Icon": "ms-appx:///Assets/Template_Blank.svg",
```
[the remark this answers is not in the document]
Comment (nickrandolph) [reply src/Uno.Templates/content/unoapp/.template.config/TemplateWizard.json:65]: it's be good if these assets were in the nuget package, and these paths updated to be a relative path

[the remark this answers is not in the document]
Comment (Takoooooo) [reply src/Uno.Templates/content/unoapp/.template.config/TemplateWizard.json:65]: Understood, will do

[the remark this answers is not in the document]
Comment (Takoooooo) [reply src/Uno.Templates/content/unoapp/.template.config/TemplateWizard.json]: Updated the paths to relative paths and included the assets which are linked in the TemplateWizard.json

**Review by dansiegel — APPROVED** [review]

File: src/Uno.Templates/content/unoapp/.template.config/Assets/Application.ID.svg
Comment (nickrandolph) [inline src/Uno.Templates/content/unoapp/.template.config/Assets/Application.ID.svg:1]: I think there should be dark theme versions of these files?

Comment (dansiegel) [reply src/Uno.Templates/content/unoapp/.template.config/Assets/Application.ID.svg:1]: @Takoooooo we can go ahead and merge this PR and I've marked this for auto-merge so it should merge when the build is complete. Can you please open an issue to follow up on @nickrandolph's feedback as a separate action item that can be part of a new PR.

Comment (Takoooooo) [reply src/Uno.Templates/content/unoapp/.template.config/Assets/Application.ID.svg:1]: @dansiegel I think i can solve that problem as the part of this PR, will address in ten minutes

**Review by nickrandolph — APPROVED** [review]

**Pull request merged**